pages/decadal.vue

```diff
@@ -1,36 +1,88 @@
 <template>
-  <div class="h-full w-full flex flex-wrap gap-4 m-8">
-    <div
-      v-for="dataset in datasets"
-      :key="dataset.title"
-      class="border-4 p-4 prose"
-    >
-      <h2>{{ dataset.title }}</h2>
-      <ul>
-        <li>Contact: <span v-for="person in dataset.contact" :key="person.name"><a target="blank" :href="getOrcid(person)">{{ person.name }}</a>, </span></li>
-        <li>License: {{ dataset.license }}</li>
-        <li>Format: {{ dataset.format }}</li>
-        <li>Data access: <a :href="dataset.doi" target="blank">{{ dataset.doi }}</a></li>
-      </ul>
-      <h3>Description</h3>
-      <nuxt-content :document="dataset" />
+  <div class="h-full w-full">
+    <!-- breadcrumbs -->
+    <div class="m-4 ml-9 text-lg">
+      <Breadcrumbs :crumbs="crumbs" />
+    </div>
+    <!-- brief summary -->
+    <p class="m-2 ml-9 text-lg">
+      Select a tab and explore the available decadal forecasts.
```

Comment:
```suggestion
      Select a tab and explore the available decadal predictions.
```

to avoid confusion between forecast/hindcast

Reply:
Corrected!